tests/conftest.py

```diff
@@ -578,3 +584,49 @@ def project_context(project: str | Path, in_place: bool = False) -> Iterator[Pat
             yield path
 
     return project_context
+
+
+@pytest.fixture()
+def temp_repo(tmp_path: Path) -> TempRepoFixture:
```

Comment:
This fixture is only used in `test_backend` and `test_system`, isn't it? Then, we can put it in a `conftest.py` in `tests/vcs/git`.

Reply:
Yes. However, it will interfere with Ruff's rules as ruff will be removed as an "unused import".
https://github.com/astral-sh/ruff/issues/3295
https://github.com/astral-sh/ruff/issues/8796

We can mark the import with `noqa` but I wasn't sure it was the best approach

Reply:
We do not have to import a fixture from a `conftest.py` to use it or do you mean something else?

Reply:
ah, sure, moved to `tests/vcs/git/conftest.py`